README.md

```diff
@@ -14,6 +14,15 @@ Lovely is a lua injector which embeds code into a [LÖVE 2d](https://love2d.org/
 
 ### Mac
 
+#### Bundle Installation (Easy)
+1. Download the [latest release](https://github.com/ethangreen-dev/lovely-injector/releases) for Mac. If you have an M-series CPU (M1, M2, etc.) then this will be `lovely-aarch64-apple-darwin-installer.tar.gz`. If you have an Intel CPU then it will be `lovely-x86_64-apple-darwin-installer.tar.gz`
```

Comment:
This step should explicitly mention the installer variant, not just any one of the mac releases. Some people won't get it otherwise.